pyproject.toml

```diff
@@ -4,7 +4,6 @@ exclude = [
   "tools/cpplint.py",
   "tools/gyp",
   "tools/inspector_protocol",
-  "tools/eslint/node_modules",
```

Comment:
This should be reverted. Ruff should exclude node_modules, even if we remove it. since it will be there for almost all developers.